### PR TITLE
combo isBundleRequestAuthorized

### PR DESCRIPTION
**Pull request by ssvegaraju**
Issue #, if available:

Description of changes:
Added combo tests and output snapshot for isBundleRequestAuthorized method.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/comboTest/isBundleRequestAuthorized-combo.test.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
+import { AuthorizationBundleRequest } from 'fhir-works-on-aws-interface';
+import { SMARTHandler } from '../smartHandler';
+import * as testStubs from './testStubs';
+import TestCaseUtil, { BaseCsvRow } from './testCaseUtil.test';
+import { convertNAtoUndefined } from './testStubs';
+
+interface CsvRow extends BaseCsvRow {
+    fhirServiceBaseUrl: string;
+    operation: string;
```
Comment (Bingjiling) [inline src/comboTest/isBundleRequestAuthorized-combo.test.ts]: operation is in BaseCsvRow

File: src/comboTest/testStubs.ts [hunk 2]
```diff
@@ -210,3 +211,42 @@ export const getResourceType = (resourceBodyDescription: ResourceBodyDescription
...
 };
+
+export const generateBundle = (): BatchReadWriteRequest[] => {
+    return [
+        {
+            operation: 'create',
+            resourceType: 'Observation',
+            id: validPatientObservation.id,
+            resource: validPatientObservation,
+            references: [
```
Comment (Bingjiling) [inline src/comboTest/testStubs.ts:223]: Where is the references field generated?

File: src/comboTest/isBundleRequestAuthorized-combo.test.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        authZConfig,
+        testStubs.apiUrl,
+        '4.0.1',
+        undefined,
+        undefined,
+        false,
+    );
+    test.each(testCases)('CASE: %s', async (testCaseString, testCase) => {
+        let testResult: any;
+        const authZHandler = testCase.isUserScopeAllowedForSystemExport
```
Comment (Bingjiling) [inline src/comboTest/isBundleRequestAuthorized-combo.test.ts]: I don't think `isUserScopeAllowedForSystemExport` is in the input csv for this test, we can remove it just to make it cleaner?

File: src/comboTest/testStubs.ts (hunk 2 above)
Comment (ssvegaraju) [reply src/comboTest/testStubs.ts:223]: References are generated [in the routing package](https://github.com/awslabs/fhir-works-on-aws-routing/blob/mainline/src/router/bundle/bundleParser.ts#L328), i used that as an example

File: src/comboTest/isBundleRequestAuthorized-combo.test.ts (hunk 3 above)
Comment (ssvegaraju) [reply src/comboTest/isBundleRequestAuthorized-combo.test.ts]: Yup, looks like i missed this and the first comment as hold overs from copying a previous test as a base. Thanks for catching that!

File: src/comboTest/testStubs.ts (hunk 2 above)
Comment (Bingjiling) [reply src/comboTest/testStubs.ts:223]: Got it! Do you mind adding a comment here with the link?

**Review by Bingjiling — COMMENTED** [review]
LGTM. Ready to merge once we finish reviewing the results.

**Review by kcadette — COMMENTED** [review]
LGTM!

**Review by kcadette — APPROVED** [review]

**Pull request merged**